ecma48/parser.go

```diff
@@ -75,7 +75,9 @@ func (p *Parser) Parse(input *bufio.Reader, output chan<- Output) error {
 	rand.Seed(time.Now().Unix())
 
 	go func() {
-		defer recover()
+		defer func() {
+			recover()
+		}()
```

Comment:
Huh, I thought these are identical?

Reply:
https://lgtm.com/rules/1511884018877/ My the looks of it, it seems that `defer recover()` doesn't catch panics, but wrapping it into a function does

Reply:
Oh wow, that's crazy. Thanks for the heads up!